packages/func/src/executors/start/executor.ts

```diff
@@ -9,7 +9,48 @@ const executor: Executor<StartExecutorSchema> = async (options, context) => {
   if (success) {
     const cwd = context.workspace?.projects[context.projectName].root;
 
-    execSync(`func start --port ${options.port}`, {
+    const {
+      port,
+      cors,
+      corsCredentials,
+      timeout,
+      useHttps,
+      cert,
+      password,
+      languageWorker,
+      noBuild,
+      enableAuth,
+      functions,
+      verbose,
+      dotnetIsolatedDebug,
+      enableJsonOutput,
+      jsonOutputFile,
+    } = options;
+
+    const args = [
+      { flag: '--port', value: port },
+      { flag: '--cors', value: cors },
+      { flag: '--cors-credentials', value: corsCredentials },
+      { flag: '--timeout', value: timeout },
+      { flag: '--useHttps', value: useHttps },
+      { flag: '--cert', value: cert, condition: useHttps },
+      { flag: '--password', value: password, condition: cert },
+      { flag: '--language-worker', value: languageWorker },
+      { flag: '--no-build', value: noBuild },
+      { flag: '--enableAuth', value: enableAuth },
+      { flag: '--functions', value: functions },
+      { flag: '--verbose', value: verbose },
+      { flag: '--dotnet-isolated-debug', dotnetIsolatedDebug },
+      { flag: '--enable-json-output', value: enableJsonOutput },
+      { flag: '--json-output-file', value: jsonOutputFile, condition: enableJsonOutput },
+    ];
+
+    const command = args
+      .filter(arg => arg.value && (!arg.condition || arg.condition))
```

Comment:
I might be mistaken, but I think you've meant to check if the condition is unset or the condition is met, right?
In that case I would suggest to check if `arg.condition === undefined`. Otherwise, the condition property will not affect the filter at all.